src/jayashri/A20_SumOfDigits.java

```diff
@@ -0,0 +1,22 @@
+package jayashri;
+
+public class A20_SumOfDigits {
+	
+	String str = "tech1noc9red3its";
+	int sum = 0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.